### DISCUSSION TITLE
Alignment of text in an element

### DISCUSSION THREAD

**Discussion by bipulsarma** [structurizr/ui#56] [Q&A]
Is there a way to align text within an element. Aligning the name of an element center or top etc. If the size of the element is 100 or below, the text appears floating near the top. 
![structurizr-SystemLandscape](https://github.com/structurizr/lite/assets/9095901/efc118c9-0409-43dd-9ea6-e108549f552e)

**Comment by mohits** [comment]
Looking at the styling capabilities, it seems like you can't do that - https://docs.structurizr.com/dsl/language#element-style

**Comment by simonbrowndotje** [comment]
I've just made a change that should make this better ... it will be in the next release.
